doc/user/content/integrations/_index.md

```diff
@@ -202,23 +202,23 @@ The level of support for these tools will improve as we extend the coverage of `
+| Metabase           | {{< supportLevel beta >}}        | Connect using the [PostgreSQL database driver](https://www.metabase.com/docs/latest/administration-guide/databases/postgresql.html). See the [Metabase integration page](/integrations/metabase/) for more details.                                                                                 | [](#notify) |
+| Superset           | {{< supportLevel alpha >}}       | Connect using the [PostgreSQL database driver](https://superset.apache.org/docs/databases/postgres/).                                                                                                                                                                                               | [](#notify) |
+| Preset             | {{< supportLevel alpha >}}       | Connect using the [PostgreSQL database driver](https://superset.apache.org/docs/databases/postgres/).                                                                                                                                                                                               | [](#notify) |
+| Looker             | {{< supportLevel alpha >}}       | Connect using the [PostgreSQL database driver](https://superset.apache.org/docs/databases/postgres/).                                                                                                                                                                                               | [](#notify) |
+| Google Data Studio | {{< supportLevel alpha >}}       | Connect using the [PostgreSQL database driver](https://support.google.com/datastudio/answer/7288010?hl#how-to-connect-to-postgresql&zippy=%2Cin-this-article).                                                                                                                                      | [](#notify) |
+| Tableau            | {{< supportLevel alpha >}}       | Connect using the [PostgreSQL database driver](https://help.tableau.com/current/pro/desktop/en-us/examples_postgresql.htm). Use the JDBC-based driver, which is included by default on Windows but requires an extra [driver download](https://www.tableau.com/support/drivers) on macOS and Linux. | [](#notify) |
+| Microsoft Power BI | {{< supportLevel researching >}} | Not supported yet. Subscribe via "Notify Me" to register interest.                                                                                                                                                                                                                                  | [](#notify) |
+| Mode Analytics     | {{< supportLevel researching >}} | Not supported yet. Subscribe via "Notify Me" to register interest.                                                                                                                                                                                                                                  | [](#notify) |
```

Comment:
```suggestion                                                                                                                                                                                                                       
```

Reply:
Mode was acquired by Thoughtspot a couple of weeks ago.

Reply:
Thanks, added.